tensorboard/plugins/custom_scalar/tf_custom_scalar_dashboard/tf-custom-scalar-margin-chart-card.html

```diff
@@ -453,6 +450,9 @@
         // Clear the data series when the tag filter changes.
         '_refreshDataSeries(_tagFilter)',
       ],
+      listeners: {
+        'dom-change': '_matchListEntryColorUpdated',
```

Comment:
It's unfortunate that we would have to
- Rely on the fact that <dom-repeat> is on the path of a dom-change event (I can't find in Polymer docs that this is intended)
- QuerySelect the list items

This is all because we cannot dynamically build styles, so this approach is to manipulate elements directly.  I'd like to propose a still-hacky change: Can we

- store the seriesName on the match-list-entry itself
- have an observer listen to `_seriesName` changes, then select the list entries and update color using the prop/attribute on the element?
Then at least we aren't tied to dom-repeat itemForElement or dom-change.

Reply:
No, we cannot. You cannot guarantee the presence of the DOM when the observer is called. The presence of DOM is predicated on two dom-ifs and one dom-repeat.

Reply:
I see.  I'll have to remember about 'dom-change' for the future :/

This works, but is hacky, let's migrate to Angular as soon as we can.

Reply:
Just so we are on the same page, the `dom-change` is documented here: 
https://polymer-library.polymer-project.org/2.0/api/elements/Polymer.DomRepeat#event-dom-change

We rely on `path` because inside shadow DOM, `target` becomes very weird and is not useful for our use case. Also see, https://polymer-library.polymer-project.org/2.0/api/elements/Polymer.DomIf#event-dom-change